invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -1,402 +1,714 @@
-          "additionalProperties": false,
-          "properties": {
-            "reference_string": {"type": "string"},
-            "identifier": {"type": "string"},
-            "scheme": {"type": "string"}
           }
-        }
-      },
-      "related_identifiers": {
-        "type": "array",
-        "uniqueItems": true,
-        "minItems": 0,
-        "items": {
+        },
+
+        "funding": {
+          "type": "array",
+          "items": {
             "type": "object",
+            "additionalProperties": false,
             "properties": {
-              "identifier": {"type": "string"},
-              "scheme": {"type": "string"},
-              "relation_type": {"type": "string"},
-              "resource_type": {
+              "funder": {
                 "type": "object",
                 "additionalProperties": false,
                 "properties": {
-                  "type": {"type": "string"},
-                  "subtype": {"type": "string"}
-                },
-                "required": ["type"]
+                  "name": {"type": "string"},
+                  "identifier": {"type": "string"},
+                  "scheme": {"type": "string"}
+                }
+              },
+              "award": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "title": {"type": "string"},
+                  "number": {"type": "string"},
+                  "identifier": {"type": "string"},
+                  "scheme": {"type": "string"}
+                }
               }
-            },
-            "required": ["identifier", "scheme", "relation_type"]
-        }
-      },
-      "resource_type": {
-        "type": "object",
-        "additionalProperties": false,
-        "description": "Record resource type.",
-        "properties": {
-          "type": {
-            "default": "publication",
-            "description": "General resource type.",
-            "type": "string"
-          },
-          "subtype": {
-            "description": "Specific resource type.",
-            "type": "string"
+            }
           }
         },
-        "required": [
-          "type"
-        ]
-      },
-      "subjects": {
-        "type": "array",
-        "uniqueItems": true,
-        "minItems": 0,
-        "items": {
+
+        "references": {
+          "type": "array",
+          "minItems": 0,
+          "items": {
             "type": "object",
+            "additionalProperties": false,
             "properties": {
-                "subject": {"type": "string"},
-                "identifier": {"type": "string"},
-                "scheme": {"type": "string"}
-            },
-            "required": ["subject"]
+              "reference_string": {"type": "string"},
+              "identifier": {"type": "string"},
+              "scheme": {"type": "string"}
+            }
+          }
         }
+      }
+    },
+
+    "metaext": {
```

Comment:
Search will be needed on `metadata` and `metaext`, we need to change this on the search obj.

Reply:
all flattened.